include/rocksdb/compaction_policy.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#pragma once
+
+#include <memory>
+
+#include "rocksdb/listener.h"
+
+namespace rocksdb {
+
+struct CompactionInfo {
+  int output_level;
+};
+
+struct CompactionOutputInfo {
+  Slice next_key;
+  uint64_t current_output_file_size;
+};
+
+class CompactionPolicy {
```

Comment:
The name of `ComapctionPolicy` is much boarder than what it does. Can we be specific, like EndCompactionOutputFilePolicy or something like that?